### ISSUE TITLE
Development - Set up gcds-fonts repo

### ISSUE THREAD

**Issue by melaniebmn** [cds-snc/design-gc-conception#988]
Icon font
- [x] Add font for icon font
- [x] Add EN readme documentation for icon font
- [x] Add FR readme documentation for icon font
- [x] Add [this OFL](https://docs.google.com/document/d/1mHhuaTlAJCt9qKIh0ewokgQCKu0UDzxiwDYnlgSY8WI/edit?tab=t.0) license
- [x] Legal: Change the license filename from “OFL.txt” to “LICENSE.txt”

Lato font
- [x] Add font for Lato font
- [x] Add EN readme documentation for Lato font
- [x] Add FR readme documentation for Lato font
- [x] Legal: Change the license filename from “OFL.txt” to “LICENSE.txt”

Noto Sans font
- [x] Add font for Noto Sans font
- [x] Add EN readme documentation for Noto Sans font
- [x] Add FR readme documentation for Noto Sans font
- [x] Legal: Change the license filename from “OFL.txt” to “LICENSE.txt”

Noto Sans Mono font
- [x] Add font for Noto Sans Mono font
- [x] Add EN readme documentation for Noto Sans Mono font
- [x] Add FR readme documentation for Noto Sans Mono font
- [x] Add OFL for Noto Sans Mono font
- [x] Legal: Change the license filename from “OFL.txt” to “LICENSE.txt”

General
- [x] Create gcds-fonts repo
- [x] Add EN readme documentation for repo
- [x] Add FR readme documentation for repo
- [x] Add release please to create releases
- [x] Add npm publish workflow to publish
- [x] Add upload to CDN workflow to upload files to our CDN S3 bucket
- [x] Add Github example pages

Legal updates
- [x] Remove any references to MIT license
- [x] Make it clear what licenses each font is released under - add "License" column to fonts table that links to each license
- [x] Change the license filename from “OFL.txt” to “LICENSE.txt”

**Comment by melaniebmn** [comment]
@daine here is the ticket that has the Github action work in it.

**Comment by melaniebmn** [comment]
Update: Icon font readme sent for translation on July 11th, 2024.

**Comment by melaniebmn** [comment]
Update: Noto Sans + Noto Sans Mono font readme sent for translation on July 15th, 2024.

**Comment by melaniebmn** [comment]
Note: reviewing the icon OFL with Sam and Nisa next week and also getting feedback from ESDC legal.

**Comment by melaniebmn** [comment]
Note: reviewing the icon OFL with Sam and Nisa next week and also getting feedback from ESDC legal.

**Comment by adorayi** [comment]
Needs an update @melaniebmn

**Comment by melaniebmn** [comment]
Update: Checked with Nisa about the legal review for the fonts and she said, she hasn't heard back yet from ESDC legal so this remains blocked from release for now.

**Comment by samburtonCDS** [comment]
Advice received from ESDC legal re: OFL license, [summary and decisions/action items here](https://docs.google.com/document/d/1yE-ySiFse8Q3af8k_c5jdJHh4edC4YXC5Yq_pMphMhU/edit) - meeting booked for Oct 15 to discuss

**Comment by samburtonCDS** [comment]
Next steps & decisions from our meeting today are [in this document.](https://docs.google.com/document/d/1yE-ySiFse8Q3af8k_c5jdJHh4edC4YXC5Yq_pMphMhU/edit?tab=t.0) 

TL;DR:
**GC Design System’s use of existing open source fonts**
- @Daine @Melanie Boeckmann will lead on updating the GCDS Font repo to remove references to the MIT license and make it clear what license each font is released under
- @Sam B (she/elle) @Nisa Malli will follow up with ESDC legal to request they lead a privacy review of Google Fonts; no change needed to the product at this time

**GC Design System’s in-house icon font**
- @Melanie Boeckmann @Daine @abdul we are good to release our in-house icon font under the reserved font name (RFN) "GCDS icons" using [this OFL](https://docs.google.com/document/d/1mHhuaTlAJCt9qKIh0ewokgQCKu0UDzxiwDYnlgSY8WI/edit)
- @Daine @Melanie Boeckmann when you go to release the icon font, I suggest you review the [How do I apply the OFL to my font?](https://openfontlicense.org/how-to-apply-the-ofl/) guidance, as there are a few other steps in there!
- @Sam B (she/elle) @Nisa Malli will follow up with ESDC legal to (a) get more clarity on if a French translation should be provided and (b) better understand the "moral rights" waiver they suggested, especially in the context of open source software

**Comment by daine** [comment]
@samburtonCDS for the license file, could we have it as LICENSE instead of LICENSE.txt? We have changed it from OFL.txt to LICENSE.txt, but I'm seeing now that github doesn't pick it up and show the license preview. Here's an example

<img width="1439" alt="Screenshot 2024-10-25 at 8 44 36 AM" src="https://github.com/user-attachments/assets/8358499a-1894-42af-b734-cb059ab45d59">


Here's where you can see them in action for comparison


- Link to one of our current licenses:
https://github.com/cds-snc/gcds-fonts/blob/main/fonts/noto-sans/LICENSE.txt

- Link to an open source font license showing the license info banner:
https://github.com/githubnext/monaspace/blob/main/LICENSE

**Comment by adorayi** [comment]
@samburtonCDS please review the latest comments and provide feedback.
